skycam_data_collection_node:
  ros__parameters:
    hz_rate: 0.5
    output_dir: "/home/williamfbx/Desktop/data"
    max_time_diff_sec: 1.0
    mode: "train"